mmeds/server.py

```diff
@@ -855,6 +855,15 @@ def run_analysis(self, access_code, analysis_method, config, runOnNode=None):
             if runOnNode and not check_privileges(self.get_user(), self.testing):
                 raise err.PrivilegeError('Only users with elevated privileges may run analysis directly')
 
+            with Database(testing=self.testing, owner=self.get_user()) as dbase:
```

Comment:
```suggestion
            with Database(testing=self.testing, owner=self.get_user()) as db:
```
generally `dbase` would be a fine variable name but I've been using `db` in this type of code block and I think it's good to be consistent throughout the project